docs/content.zh/docs/deployment/resource-providers/standalone/kubernetes.md

```diff
@@ -338,18 +366,19 @@ data:
     scheduler-mode: reactive
     execution.checkpointing.interval: 10s
   log4j-console.properties: |+
-    # This affects logging for both user code and Flink
+    # 如下配置会同时影响用户代码和 Flink 的日志行为
     rootLogger.level = INFO
     rootLogger.appenderRef.console.ref = ConsoleAppender
     rootLogger.appenderRef.rolling.ref = RollingFileAppender
 
-    # Uncomment this if you want to _only_ change Flink's logging
+    # 如果只想改变 Flink 的日志行为可以取消如下的注释符
     #logger.flink.name = org.apache.flink
     #logger.flink.level = INFO
 
-    # The following lines keep the log level of common libraries/connectors on
-    # log level INFO. The root logger does not override this. You have to manually
-    # change the log levels here.
+
+    # 下面几行将公共 libraries 或 connectors 的日志级别保持在 INFO 级别。
+    # root logger 的配置不会覆盖此处配置。
+    # 必须手动修改这里的日志级别。
```

Comment:
here, too.

Reply:
Ok, take your advice. It has been fixed.